pkg/reconciler/route/reconcile_resources.go

```diff
@@ -227,3 +248,30 @@ func (c *Reconciler) reconcileTargetRevisions(ctx context.Context, t *traffic.Co
 	}
 	return eg.Wait()
 }
+
+func serializeRollout(ctx context.Context, r *traffic.Rollout) string {
+	sr, err := json.Marshal(r)
+	if err != nil {
+		// This must not never happen in the normal course of things.
```

Comment:
super nit but I think this is a double negative and you actually want to say
```suggestion
		// This should never happen in the normal course of things.
```

Reply:
yep :) thanks.